activerecord/lib/active_record/connection_adapters/sqlite3/schema_definitions.rb

```diff
@@ -4,6 +4,18 @@ module ActiveRecord
   module ConnectionAdapters
     module SQLite3
       class TableDefinition < ActiveRecord::ConnectionAdapters::TableDefinition
+        attr_reader :strict_table
+
+        def initialize(
+          conn,
+          name,
+          strict_table: false,
+          **options
+        )
```

Comment:
These parameters should all be on one line - we don't usually do this style in Rails.